### DISCUSSION TITLE
Code highlight ?

### DISCUSSION THREAD

**Discussion by YoannBuzenet** [ianstormtaylor/slate#5140] [General]
Hello,

Is there an out-of-the-box plugin to make code highlighting in slate ?

Thank you

**Comment by dylans** [comment]
Plate (Plugins for Slate) https://github.com/udecode/plate has one that leverages Prism.

**Reply by YoannBuzenet** [reply]
Hey, I can't find it in their doc nor their website. Would you have an URL for me to check ?

**Reply by dylans** [reply]
It's the CodeBlock plugin. I'm on the slowest airplane wifi imaginable so if you cannot find it based on that let me know and I'll provide a link tomorrow.

**Reply by dylans** [reply]
https://github.com/udecode/plate/tree/main/packages/nodes/code-block

**Reply by YoannBuzenet** [reply]
Thank you, I'll have a look at this.
